test/acceptance/ReactRefreshLogBox.test.js

```diff
@@ -812,13 +812,11 @@ test('css syntax errors', async () => {
   await session.patch('index.module.css', `.button {`)
   expect(await session.hasRedbox(true)).toBe(true)
   const source = await session.getRedboxSource()
-  expect(source).toMatchInlineSnapshot(`
-    "./index.module.css:1:1
-    Syntax error: Unclosed block
-
-    > 1 | .button {
-        | ^"
-  `)
+  expect(source).toMatch('./index.module.css:1:1')
```

Comment:
What was the difference here that required updating it to no longer use matchInlineSnapShot 🤔

Reply:
`postcss-loader` now emits an error message which includes the file path (since [this commit](https://github.com/webpack-contrib/postcss-loader/commit/4b44e01a323aa9d2c0965e92c1796158cb36ce5a) on their end).